### DISCUSSION TITLE
Compare the parameters after each SI

### DISCUSSION THREAD

**Discussion by ledinhduy281** [HewlettPackard/swarm-learning#170] [Q&A]
I am implementing a function that evaluates the performance of SN nodes participating in training. I have 2 questions:
1. Assuming there is a node with poor computation, is there any way I can compare the parameters between nodes in the end of each SI?
2. With the finding of the poorly calculated node, can I adjust the weights of the SN nodes by resetting the NodeWeightage parameter in SwarmCallback for the next SI?

Hope to get your reply. Thank you!

**Comment by iArpanPatel** [comment]
Dynamic adjustments to the node weightage is not supported. 
If you can elaborate on your use case, we can explore on alternatives.

**Reply by iArpanPatel** [reply]
@ledinhduy281 any update on this discussion?
